src/fsharp/fscopts.fs

```diff
@@ -444,7 +444,6 @@ let testFlag tcConfigB =
 // not shown in fsc.exe help, no warning on use, motiviation is for use from VS
 let vsSpecificFlags (tcConfigB: TcConfigBuilder) = 
   [ CompilerOption("vserrors", tagNone, OptionUnit (fun () -> tcConfigB.errorStyle <- ErrorStyle.VSErrors), None, None);
-    CompilerOption("validate-type-providers", tagNone, OptionUnit (fun () -> tcConfigB.validateTypeProviders <- true), None, None);
```

Comment:
set as deprecated and noop? Because is a minor version (oob), no breaking changes
so if someone use it, get the warning but no error (build failed)
Next version (master?), cleanup and remove all deprecated commands?